install/linux/docker-ee/ubuntu.md

```diff
@@ -247,13 +247,20 @@ To upgrade Docker Engine - Enterprise, follow the steps below:
 ### Install from a package
 
 If you cannot use Docker's repository to install Docker Engine - Enterprise, you
-can download the `.deb` file for your release and install it manually. You need
-to download a new file each time you want to upgrade Docker Engine - Enterprise.
+can download the `.deb` files for your release and install them manually. You
+need to download a new file or set of files each time you want to upgrade Docker
+Engine - Enterprise.
 
 1.  Go to the Docker Engine - Enterprise repository URL associated with your
     trial or subscription in your browser. Go to
-    `ubuntu/x86_64/stable-<VERSION>` and download the `.deb` file for the
-    Docker Engine - Enterprise version and architecture you want to install.
+    `ubuntu/dists/<distribution>/pool/stable-<VERSION>` and download the `.deb`
+    file(s) for the Docker Engine - Enterprise version and architecture you want
+    to install.
+
+    > **Note:**
+    > Starting with 19.03, you will download three `.deb` files. They
```

Comment:
"Will download" makes it sound like 19.03 is a future release. I'd rephrase to something like "you need to" or "you have to".